src/airflow/dags/airnow_bam_data.py

```diff
@@ -119,9 +119,9 @@ def update_latest_data_topic(data: pd.DataFrame):
     extracted_bam_data = extract_bam_data()
     processed_bam_data = process_data(extracted_bam_data)
     send_to_bigquery(processed_bam_data)
-    update_latest_data_table(processed_bam_data)
-    update_latest_data_topic(processed_bam_data)
-    send_measurements_to_api(processed_bam_data)
+    # update_latest_data_table(processed_bam_data)
+    # update_latest_data_topic(processed_bam_data)
+    # send_measurements_to_api(processed_bam_data)
```

Comment:
Temporarily disabling sending US Embassy data to API.

Reply:
Alright. @NoahNsimbe 
Is there anyway I could be of help in resolving this issue from an API perspective?

Reply:
I think we discussed this @Baalmart .... The unique fields for US Embassy needs to updated so that we can easily map the devices and sites to air quality readings. I don't know the progress of this task.

Reply:
Hi @NoahNsimbe , this was addressed by this[ recently merged PR](https://github.com/airqo-platform/AirQo-api/pull/1190)?

In case [that merged PR](https://github.com/airqo-platform/AirQo-api/pull/1190) addresses all the technical requirements, is the pending work mostly "data entry" in nature?